jsonsch/util.go

```diff
@@ -10,7 +10,7 @@ import (
 func buildSchema(fromValue interface{}, dst *interface{}, params *FromExampleParams) error {
 	switch v := fromValue.(type) {
 	case nil:
-		*dst = NewNull()
+		*dst = NewNull(params)
```

Comment:
`schema infer --graphql` works like this right now

1. Read incoming JSON/YAML/TOML, etc.
2. Create an `jsonsch.Schema` object in memory
3. Call the method `js.ToGraphQLSchema()` to convert it to a `graphqlsch.Schema` object
4. Serialize the `graphqlsch.Schema` object

All the logic of this PR looks correct, but it's in part 2 (from the numbering above^) of the codebase. The lines we need to modify are here:

call to handlePrimitive: https://github.com/Confbase/schema/blob/master/jsonsch/jsonsch.go#L32

actual line that rejects null values in handlePrimitive: https://github.com/Confbase/schema/blob/master/jsonsch/jsonsch.go#L96

call to handleArraySchema: https://github.com/Confbase/schema/blob/master/jsonsch/jsonsch.go#L44

actual line that rejects null values: https://github.com/Confbase/schema/blob/master/jsonsch/jsonsch.go#L140

The codebase has zero documentation right now, so no question is a dumb question.